src/lib/features/project/project-owners-read-model.test.ts

```diff
@@ -4,10 +4,53 @@ import { type IUser, RoleName, type IGroup } from '../../types';
 import { randomId } from '../../util';
 import { ProjectOwnersReadModel } from './project-owners-read-model';
 
+const mockProjectWithCounts = (name: string) => ({
+    name,
+    id: name,
+    description: '',
+    featureCount: 0,
+    memberCount: 0,
+    mode: 'open' as const,
+    defaultStickiness: 'default' as const,
+    staleFeatureCount: 0,
+    potentiallyStaleFeatureCount: 0,
+    avgTimeToProduction: 0,
+});
+
 describe('unit tests', () => {
-    test('maps owners to projects', () => {});
+    test('maps owners to projects', () => {
+        const projects = [{ name: 'project1' }, { name: 'project2' }] as any;
+
+        const owners = {
+            project1: [{ ownerType: 'user' as const, name: 'Owner Name' }],
+            project2: [{ ownerType: 'user' as const, name: 'Owner Name' }],
+        };
+
+        const projectsWithOwners = ProjectOwnersReadModel.addOwnerData(
+            projects,
+            owners,
+        );
+
+        expect(projectsWithOwners).toMatchObject([
+            { name: 'project1', owners: [{ name: 'Owner Name' }] },
+            { name: 'project2', owners: [{ name: 'Owner Name' }] },
+        ]);
+    });
+
     test('returns "system" when a project has no owners', async () => {
-        // this is a mapping test; not an integration test
+        const projects = [{ name: 'project1' }, { name: 'project2' }] as any;
+
+        const owners = {};
+
+        const projectsWithOwners = ProjectOwnersReadModel.addOwnerData(
+            projects,
+            owners,
+        );
+
+        expect(projectsWithOwners).toMatchObject([
+            { name: 'project1', owners: [{ ownerType: 'system' }] },
+            { name: 'project2', owners: [{ ownerType: 'system' }] },
+        ]);
     });
 });
```

Comment:
Can be done later, but I think we should put the unit tests in a separate file? So we'll probably have `project-owners-read-model.test.ts` and `project-owners-read-model.integration.test.ts`. Feels like it jives better with what we're doing most the time.

Reply:
Fair enough 💁🏼